std_msgs/String camera

---
sensor_msgs/Image frame
